tests/nativeapp/test_post_deploy_for_app.py

```diff
@@ -56,35 +57,49 @@ def test_sql_scripts(
     mock_cli_ctx,
     mock_execute_queries,
     mock_execute_query,
-    project_directory,
+    temp_dir,
 ):
     mock_conn.return_value = MockConnectionCtx()
     mock_cli_ctx.return_value = {
         "ctx": {"native_app": {"name": "myapp"}, "env": {"foo": "bar"}}
     }
-    with project_directory("napp_post_deploy") as project_dir:
-        processor = _get_run_processor(str(project_dir))
+    post_deploy_1 = dedent(
+        """\
+            -- app post-deploy script (1/2)
 
-        processor.execute_app_post_deploy_hooks()
+            select myapp;
+            select bar;
+            """
+    )
+    post_deploy_2 = "-- app post-deploy script (2/2)\n"
+
+    ProjectV11Factory(
+        pdf__native_app__artifacts=[{"src": "app/*", "dest": "./"}],
+        pdf__native_app__name="myapp",
+        pdf__native_app__application__post_deploy=[
+            {"sql_script": "scripts/app_post_deploy1.sql"},
+            {"sql_script": "scripts/app_post_deploy2.sql"},
+        ],
+        pdf__env__foo="bar",
+        pdf__env__package_foo="package_bar",
```

Comment:
do we need these for this specific test?

Reply:
nope! Great catch made possible by the new way of writing this test!! 🕺